include/bluetooth/mesh/main.h

```diff
@@ -105,6 +111,14 @@ struct bt_mesh_prov {
 	/** Out of Band information field. */
 	bt_mesh_prov_oob_info_t oob_info;
 
+	/** Public Key type */
+	enum bt_mesh_pub_key_type_t pub_key_type;
+
+	/** Pointer to Public Key for static OOB Public Key */
+	const uint8_t *public_key;
+	/** Pointer to Private Key for static OOB Public Key */
+	const uint8_t *private_key;
```

Comment:
This is different from the provisioner side of the OOB mechanism, which uses a function. It would be nice to use the same pattern (if not the same mechanism), but I don't see any obvious solution to this. Any suggestions?

Reply:
I guess in case of provisioner the idea is that public key changes from provisionee to provisionee, while for provisionee it's kind of static, which doesn't make sense to use a function.

Reply:
That's true, yeah. I might be looking for a pattern where there is none :slightly_smiling_face: